bitnami/concourse/Chart.yaml

```diff
@@ -30,4 +30,4 @@ name: concourse
 sources:
   - https://github.com/bitnami/bitnami-docker-concourse
   - https://github.com/concourse/concourse
-version: 1.0.3
```

Comment:
Please remove this change from the PR since it belongs to concourse

Reply:
Done.Please review it again.Thank you^_^